### ISSUE TITLE
Setuptools error while installing

### ISSUE THREAD

**Issue by serhangul** [luismartingarcia/protocol#16]
Hi, when I try to install using `python setup.py install`, I get this error:
```
error: Multiple top-level modules discovered in a flat-layout: ['specs', 'constants'].

To avoid accidental inclusion of unwanted files or directories,
setuptools will not proceed with this build.

If you are trying to create a single distribution with multiple modules
on purpose, you should not rely on automatic discovery.
Instead, consider the following options:

1. set up custom discovery (`find` directive with `include` or `exclude`)
2. use a `src-layout`
3. explicitly set `py_modules` or `packages` with a list of names
```

Any idea where this comes from? My guess is it is some thing related to setuptools (using version 65.5.0) but I don't know enough about it to solve this. Is there a way to install this module using a package manager like pip? Thanks.

**Comment by leytou** [comment]
You can use my forked project: https://github.com/leytou/protocolpro
This project fixes installation related issues as well as fixes #9

You can install it directly with the following command:
`pip install protocolpro`

**Comment by tan-wei** [comment]
Great work! @leytou
